### ISSUE TITLE
Accept `CCL-CCTF-WGFS` acronym as committee name

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-bipm#64]
This error is raised when compiling SI Brochure rec-values documents:

```
[relaton-bipm] WARN: Invalid acronym: `CCL-CCTF-WGFS`. Allowed values: `CCAUV`, `CCDS`, `CCE`, `CCEM`, `CCL`, `CCM`, `CCPR`, `CCQM`, `CCRI`, `CCT`, `CCTF`, `CCU`, `CGPM`, `CIPM`, `BIPM`, `CCL-CCTF`, `JCGM`, `JCRB`, `JCTLM`, `INetQI`
```

However, these documents are indeed published by `CCL-CCTF-WGFS` which is this group:

* https://www.bipm.org/en/committees/cc/cctf/wg/ccl-cctf-wgfs
* https://www.metanorma.org/author/bipm/ref/document-attributes/ 

And is also accepted in Metanorma:

<img width="663" alt="Screenshot 2024-12-16 at 11 46 13 PM" src="https://github.com/user-attachments/assets/bc55bb1f-2dd4-4e10-8565-e4884fe58bb1" />

**Comment by andrew2net** [comment]
Fixed in relaton-bipm v1.19.4 and v1.20.2

**Comment by ronaldtse** [comment]
Thank you!
